bin/src/command/mod.rs

```diff
@@ -1013,17 +998,27 @@ async fn worker_loop(
                 error!("could not decode worker message: {:?}", e);
                 break;
             }
-            Ok(message) => {
-                debug!("worker {} replied message: {:?}", id, message);
+            Ok(proxy_response) => {
+                debug!("worker {} replied message: {:?}", id, proxy_response);
                 let id = id.clone();
-                tx.send(CommandMessage::WorkerResponse { id, message })
+                if let Err(e) = command_tx
+                    .send(CommandMessage::WorkerResponse {
+                        id,
+                        message: proxy_response,
+                    })
                     .await
-                    .unwrap();
+                {
+                    error!("error sending worker response to command server: {:?}", e);
+                }
             }
         }
     }
 
-    tx.send(CommandMessage::WorkerClose { id }).await.unwrap();
+    // if the loop breaks, order the command server to close the worker
+    command_tx
+        .send(CommandMessage::WorkerClose { id })
+        .await
+        .unwrap();
```

Comment:
Could you change it to a `.with_context(...)?`

Reply:
Well this is executed in a detached thread, so I'll just log the error in place.